main.js

```diff
@@ -0,0 +1,32 @@
+function generateNumbers() {
+    let numbersOnScreen = Math.floor(Math.random() * 3) + 3;
+
+    let allNumbers = [];
```

Comment:
Since these are both variable declarations they can be next to each other, the blank line between isn't needed.